gamemodes/terrortown/gamemode/shared/hud_elements/tttinfopanel/old_ttt_info.lua

```diff
@@ -62,9 +62,14 @@ if CLIENT then
 			-- Draw health
 			local health = math.max(0, client:Health())
 			local health_y = y + margin
+			local health_string = tostring(health)
+			
+			if not GetGlobalBool("ttt_armor_classic", false) and client:Armor() > 0 then
+				health_string = health_string .. " + " .. tostring(client:Armor())
```

Comment:
sure u need to call the "tostring" function?

Reply:
yes

Reply:
why?

Reply:
without it it will throw an error because it can't concat an integer to a string

Reply:
Could you send this error?

Reply:
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!?